readthedocs/projects/models.py

```diff
@@ -259,6 +259,12 @@ class AddonsConfig(TimeStampedModel):
 
     # Link Previews
     linkpreviews_enabled = models.BooleanField(default=False)
+    linkpreviews_selector = models.CharField(
+        null=True,
+        blank=True,
+        max_length=128,
+        help_text="CSS selector for the links preview of the page. Leave it blank for auto-detect.",
```

Comment:
This needs a better description, I think? it's not clear to me what I should do with this information to actually implement it? Perhaps needs a docs link?

Reply:
I followed what we already have for the `options_root_selector` at https://github.com/readthedocs/readthedocs.org/blob/main/readthedocs/projects/models.py#L164

Do you have a suggestion for a better `help_text`?

Reply:
```suggestion
        help_text="CSS selector to select links you want enabled for link previews. Leave it blank for auto-detect all links in your main page content.",
```

Looking at the addons tests, I think something like this?

Reply:
Done in [`b390288` (#12301)](https://github.com/readthedocs/readthedocs.org/pull/12301/commits/b390288c1460e98c62f8a9c6d6b2fcc3fe246deb)